src/test/unit/specific_feature_tests/exec_toggles.js

```diff
@@ -338,67 +338,187 @@ function doesntWrapAcrossRootContainers(command) {
+        closeTag = "</" + tagName + ">";
+    manipulationTestHelper({
+        startHtml: [
+            "<ul>",
+                "<li>F" + openTag + "oo" + closeTag + "</li>",
+                "<li>" + openTag + "Another sibling" + closeTag + "</li>",
+                "<li>" + openTag + "This:<br />" +
+                    closeTag + "<span>" + openTag + "ba" + closeTag + "r</span></li>",
+            "</ul>"
+        ].join(''),
+        prepareFunc: function (wymeditor) {
+            var $body = wymeditor.$body(),
+                firstWrapper = $body.find(tagName).first()[0],
+                lastWrapper = $body.find(tagName).last()[0];
+            makeTextSelection(
+                wymeditor,
+                firstWrapper,
+                lastWrapper,
+                0,
+                2
+            );
+        },
+        manipulationFunc: function (wymeditor) {
+            wymeditor.exec(command);
+        },
+        expectedResultHtml: [
+            "<ul>",
+                "<li>Foo</li>",
+                "<li>Another sibling</li>",
+                "<li>This:<br /><span>bar</span></li>",
+            "</ul>"
+        ].join(''),
+        manipulationKeyCombo: getExecKeyboardShortcut(command),
+        testUndoRedo: true
+    });
+}
+
+test("Bolds from inside `li` to child of sibling `li`", function () {
+    wrapsFromInsideLiToChildOfSiblingLi(WYMeditor.EXEC_COMMANDS.BOLD);
+});
+
+test("Italics from inside `li` to child of sibling `li`", function () {
+    wrapsFromInsideLiToChildOfSiblingLi(WYMeditor.EXEC_COMMANDS.ITALIC);
+});
+
+test("Superscripts from inside `li` to child of sibling `li`", function () {
+    wrapsFromInsideLiToChildOfSiblingLi(WYMeditor.EXEC_COMMANDS.SUPERSCRIPT);
+});
+
+test("Subscripts from inside `li` to child of sibling `li`", function () {
+    wrapsFromInsideLiToChildOfSiblingLi(WYMeditor.EXEC_COMMANDS.SUBSCRIPT);
+});
+
+test("unwraps bold from inside `li` to child of sibling `li`", function () {
+    unwrapsFromInsideLiToChildOfSiblingLi(WYMeditor.EXEC_COMMANDS.BOLD);
 });
 
-test("Doesn't unwrap italic across root containers", function () {
-    doesntUnwrapAcrossRootContainers(WYMeditor.EXEC_COMMANDS.ITALIC);
+test("unwraps italic from inside `li` to child of sibling `li`", function () {
+    unwrapsFromInsideLiToChildOfSiblingLi(WYMeditor.EXEC_COMMANDS.ITALIC);
 });
 
-test("Doesn't unwrap superscript across root containers", function () {
-    doesntUnwrapAcrossRootContainers(WYMeditor.EXEC_COMMANDS.SUPERSCRIPT);
+test("unwraps superscript from inside `li` to child of sibling `li`", function () {
+    unwrapsFromInsideLiToChildOfSiblingLi(WYMeditor.EXEC_COMMANDS.SUPERSCRIPT);
 });
 
-test("Doesn't unwrap subscript across root containers", function () {
-    doesntUnwrapAcrossRootContainers(WYMeditor.EXEC_COMMANDS.SUPERSCRIPT);
+test("unwraps subscript from inside `li` to child of sibling `li`", function () {
+    unwrapsFromInsideLiToChildOfSiblingLi(WYMeditor.EXEC_COMMANDS.SUBSCRIPT);
```

Comment:
This is a mess. Perhaps in the commit, it is easier to see that I added some more tests.